Loop/Views/LoggedDoseView.swift

```diff
@@ -0,0 +1,399 @@
+//
+//  LoggedDoseView.swift
+//  Loop
+//
+//  Created by Pete Schwamb on 12/29/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Combine
+import HealthKit
+import SwiftUI
+import LoopKit
+import LoopKitUI
+import LoopUI
+
+
+struct LoggedDoseView: View {
```

Comment:
In terms of breaking this out as a separate view, I'm wondering if it makes sense to remove the manual glucose logic as well. My rationale for this is that the BG graph is less significant when logging doses (we're already not displaying the recommended bolus), so this would simplify the UI

Reply:
Depending on how you see this view being used in the future, I also think we may be able to get rid of the carb logic